docs/usage/channels.rst

```diff
@@ -0,0 +1,452 @@
+.. currentmodule:: litestar.channels
+
+Channels
+========
+
+**Channels** are a group of related functionalities, built to facilitate the routing
+of event streams, which for example can be used to broadcast messages to WebSocket
+clients.
+
+Channels provide:
+
+1. Independent :term:`broker` backends, optionally handling inter-process communication
+   and data persistence on demand
+2. "Channel" based subscription management
+3. Subscriber objects as an abstraction over an individualized :term:`event stream`,
+   providing background workers and managed subscriptions
+4. Synchronous and asynchronous data publishing
+5. Optional history management on a per-channel basis
+6. :doc:`WebSocket </usage/websockets>` integration, generating WebSocket route
+   handlers for an application, to handle the subscription and publishing of incoming
+   events to the connected client
+
+
+Basic concepts
+--------------
+
+Utilizing channels involves a few moving parts, of which the most important ones are:
+
+.. glossary::
+
+    event
+        A single piece of data published to, or received from a :term:`backend`, bound
+        to the channel it was originally published to
+
+    event stream
+        A stream of :term:`events <event>`, consisting of events from all the channels a
+        :term:`Subscriber` has previously subscribed to
+
+    subscriber
+        A :class:`Subscriber <.subscriber.Subscriber>`: An object wrapping an
+        :term:`event stream` and providing access to it through various methods
+
+    backend
+        A :class:`ChannelsBackend <.backends.base.ChannelsBackend>`. This object
+        manages communication between the plugin and the :term:`broker`, publishing
+        messages to and receiving messages from it. Each plugin instance is associated
+        with exactly one backend.
+
+    broker
+        Responsible for receiving and publishing messages to all connected
+        :term:`backends <backend>`; All backends sharing the same broker will have
+        access to the same messages, allowing for inter-process communication. This is
+        typically handled by a separate entity like `Redis <https://redis.io/>`_
+
+    plugin
+        The :class:`ChannelsPlugin <.plugin.ChannelsPlugin>`, a central instance
+        managing :term:`subscribers <subscriber>`, reading messages from the
```

Comment:
```suggestion
        managing :term:`subscribers <subscriber>`; reading messages from the
```

Reply:
This changes the meaning and makes it incorrect. It's a list of different functionalities, the latter items of which are not directly related to the management of subscribers.